client/src/components/CardNavegation/CardNavegation.css

```diff
@@ -0,0 +1,11 @@
+.card-nav{
+    width: 10vw;
+    height: 200px;
+    background-color: cadetblue;
+    margin-left: 50px;
+    margin-top: 50px;
+}
+.card-nav li{
```

Comment:
Needs consistent spacing

Reply:
I fixed the spacing of this css file. Hopes it looks better now